app/src/main/java/com/simplecity/amp_library/model/Song.java

```diff
@@ -210,7 +210,7 @@ public void setResumed() {
      *
      * @return the elapsed time of this track (in millis)
      */
-    private long getElapsedTime() {
+    public long getElapsedTime() {
```

Comment:
Can you please open a separate issue about testing elapsed time values? I want to make sure we don't forget this. It's still not clear to me if this will give us what we want when a user resumes playing a song, which is the start time index within the song. If not, we may need to figure out a way to track that separately.